deploy/utils.sh

```diff
@@ -166,7 +172,7 @@ fetch_ssl_from_conjur() {
     export cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
   fi
 
-  export conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
+  export conjur_pod_name="$(get_pod_name "${CONJUR_NAMESPACE_NAME}" $selector)"
```

Comment:
`$selector` should be double quoted.  Also for consistency, `"$CONJUR_NAMESPACE_NAME"` is fine here -- no need for the braces.